R/understat_teams.R

```diff
@@ -52,55 +52,57 @@ understat_team_stats_breakdown <- function(team_urls) {
 }
 
 .understat_team_stats_breakdown <- function(team_url) {
-  data_html <-
```

Comment:
the code diff looks big here. really, it's just one refactor at the top here + the addition of an `if(!is.na(data_html))` check